packages/core/useMediaQuery/index.ts

```diff
@@ -15,10 +36,10 @@ export function useMediaQuery(query: string) {
     matches.value = event.matches
   }
 
-  mediaQuery.addEventListener('change', handler)
+  matchMediaListener(mediaQuery, handler)
 
   tryOnUnmounted(() => {
-    mediaQuery.addEventListener('change', handler)
+    matchMediaListener(mediaQuery, handler)
```

Comment:
@antfu Maybe I misunderstand here something. Shouldn't the event be removed in tryOnUnmounted?

Reply:
Yeah, was a typo here, thanks